api/proto/teleport/legacy/types/types.proto

```diff
@@ -5626,6 +5626,20 @@ message PluginOktaSettings {
 
   // SSOConnectorID
   string sso_connector_id = 3;
+
+  // Import settings controls the access list import settings for Okta.
+  PluginOktaImportSettings import = 4;
+}
+
+// Defines settings for importing access lists from Okta.
+message PluginOktaImportSettings {
```

Comment:
Probably we will need also add filtering param

Reply:
We definitely will, but I don't want to hold the import up on filtering until we've got it more fully implemented.